models/DataObject/ClassDefinition/Data/Fieldcollections.php

```diff
@@ -687,13 +687,18 @@ public function classSaved($class, $params = [])
 
                     foreach ($fieldDefinition as $fd) {
                         //TODO Pimcore 11 remove method_exists call
-                        if (!$fd instanceof DataContainerAwareInterface && method_exists($fd, 'classSaved')) {
+                        if (!$fd instanceof DataContainerAwareInterface && ($fd instanceof ClassSavedInterface || method_exists($fd, 'classSaved'))) {
+                            if (!$fd instanceof ClassSavedInterface) {
+                                trigger_deprecation('pimcore/pimcore', '10.6',
+                                    sprintf('Usage of method_exists is deprecated since version 10.6 and will be removed in Pimcore 11.' .
+                                    'Implement the %s interface instead.', ClassSavedInterface::class));
+                            }
                             // defer creation
-                            $fd->classSaved($class);
+                            $fd->classSaved($class, $params);
                         }
                     }
 
-                    $definition->getDao()->classSaved($class);
+                    $definition->getDao()->classSaved($class, $params);
```

Comment:
This method have no params argument
```suggestion
                    $definition->getDao()->classSaved($class);
```